ddtrace/contrib/dbapi/__init__.py

```diff
@@ -69,7 +70,7 @@ def _trace_method(self, method, name, resource, extra_tags, *args, **kwargs):
 
             # set analytics sample rate if enabled but only for non-FetchTracedCursor
             if not isinstance(self, FetchTracedCursor):
-                s.set_tag(ANALYTICS_SAMPLE_RATE_KEY, config.dbapi2.get_analytics_sample_rate())
+                s.set_tag(ANALYTICS_SAMPLE_RATE_KEY, cfg.get_analytics_sample_rate())
```

Comment:
```suggestion
                s.set_tag(ANALYTICS_SAMPLE_RATE_KEY, self._self_config.get_analytics_sample_rate())
```

?

Reply:
Ah! I didn't push my latest change where I'd fixed this!